src/main/g8/scalafix/build.sbt

```diff
@@ -1,42 +1,61 @@
 lazy val V = _root_.scalafix.sbt.BuildInfo
+lazy val scala3InputOutputVersion = "3.0.0"
+
 inThisBuild(
   List(
     scalaVersion := V.scala213,
     crossScalaVersions := List(V.scala213, V.scala212, V.scala211),
     organization := "com.example",
     homepage := Some(url("https://github.com/com/example")),
-    licenses := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+    licenses := List(
+      "Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")
+    ),
     developers := List(
       Developer(
         "example-username",
         "Example Full Name",
         "example@email.com",
         url("https://example.com")
       )
-    ),
-    addCompilerPlugin(scalafixSemanticdb),
-    scalacOptions ++= List(
-      "-Yrangepos",
-      "-P:semanticdb:synthetics:on"
     )
   )
 )
 
 publish / skip := true
+crossScalaVersions := Nil
 
 lazy val rules = project.settings(
   moduleName := "scalafix",
   libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % V.scalafixVersion
 )
 
-lazy val input = project.settings(
-  publish / skip := true
-)
+lazy val input = project
+  .settings(
+    publish / skip := true,
+    crossScalaVersions += scala3InputOutputVersion,
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision,
+    scalacOptions ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, _)) => List("-P:semanticdb:synthetics:on")
+        case _            => Nil
+      }
+    }
+  )
 
 lazy val output = project.settings(
-  publish / skip := true
+  publish / skip := true,
+  crossScalaVersions += scala3InputOutputVersion
 )
 
+// for +test to cascade to `tests` project for scala3InputOutputVersion
```

Comment:
this trick does not work if test is called directly under the tests project, so https://scalacenter.github.io/scalafix/docs/developers/setup.html should be updated if we go that way